src/libraries/System.Formats.Tar/tests/TarFile/TarFile.CreateFromDirectory.File.Tests.cs

```diff
@@ -253,5 +255,57 @@ public void SkipRecursionIntoBaseDirectorySymlink()
 
             Assert.Null(reader.GetNextEntry());
         }
+
+        [ConditionalTheory]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        [MemberData(nameof(GetPaxAndGnuTestCaseNames))]
```

Comment:
We need another condition here confirming that the `/usr/bin/tar` binary exists. We don't know if some Unix flavors put the binary somewhere else.

Reply:
We assume /bin/ln always exists, I thought we could do the same here.
https://github.com/dotnet/runtime/blob/9ea5d2d6cdc536fbe1dacc477ef2b2dea2b4845c/src/libraries/Common/tests/System/IO/ReparsePointUtilities.cs#L94

Reply:
I guess we will know when the CI is done.

Reply:
Another option is to assume `/usr/bin` is on the $PATH in every distro we test on. Just as you're assuming here that `C:\Windows\System32` is on the Windows PATH

Reply:
Hmm, on Alpine, it's implemented by Busybox, with a link. So `/usr/bin/tar` does not exist, but `tar` should work:

```
danmoseL:~# ls -alF /bin/tar
lrwxrwxrwx    1 root     root            12 Mar 31  2021 /bin/tar -> /bin/busybox*
```

Reply:
And incidentally, the busybox tar is probably significantly different to the GNU and BSD tar's so it would be good to try it. (You can also do it locally in WSL2 of course if you want)